doc/quick-start.rst

```diff
@@ -255,6 +255,42 @@ Write this in your ``jbuild`` file:
 
 And run the tests with:
 
+.. code:: bash
+
+    jbuilder runtest
+
+Build and run multiple targets
+==============================
+
+Let's say you have 2 executable targets ``tests/hello.exe`` and ``tests/world.exe`` and you want to build and run them seamlessly. Your ``tests/jbuild`` file should look like this:
+
+.. code:: scheme
+    (jbuild_version 1)
+
+    (executables
```

Comment:
Given that onle one executable is being defined, we should use an `executable` stanza